openshift/dockerfiles/webhook.Dockerfile

```diff
@@ -0,0 +1,35 @@
+ARG GO_BUILDER=brew.registry.redhat.io/rh-osbs/openshift-golang-builder:v1.22
+ARG RUNTIME=registry.access.redhat.com/ubi9/ubi-minimal:latest@sha256:c0e70387664f30cd9cf2795b547e4a9a51002c44a4a86aa9335ab030134bf392
+
+FROM $GO_BUILDER AS builder
+
+WORKDIR /go/src/github.com/openshift-pipelines/pipelines-as-code
+COPY . .
+RUN set -e; for f in patches/*.patch; do echo ${f}; [[ -f ${f} ]] || continue; git apply ${f}; done
+ENV GODEBUG="http2server=0"
+RUN go build -mod=vendor -tags disable_gcp -v -o /tmp/pipelines-as-code-webhook \
+    ./cmd/pipelines-as-code-webhook
+
+FROM $RUNTIME
+ARG VERSION=pipelines-as-code-webhook-main
+
+ENV KO_APP=/ko-app
+
+COPY --from=builder /tmp/pipelines-as-code-webhook ${KO_APP}/pipelines-as-code-webhook
+
+LABEL \
+      com.redhat.component="openshift-pipelines-pipelines-as-code-webhook-container" \
+      name="openshift-pipelines/pipelines-pipelines-as-code-webhook-rhel8" \
+      version=$VERSION \
+      summary="Red Hat OpenShift Pipelines Pipelines as Code Webhook" \
+      maintainer="pipelines-extcomm@redhat.com" \
+      description="Red Hat OpenShift Pipelines Pipelines as Code Webhook" \
+      io.k8s.display-name="Red Hat OpenShift Pipelines Pipelines as Code Webhook" \
+      io.k8s.description="Red Hat OpenShift Pipelines Pipelines as Code Webhook" \
+      io.openshift.tags="pipelines,tekton,openshift"
+
+RUN microdnf install -y shadow-utils
+RUN groupadd -r -g 65532 nonroot && useradd --no-log-init -r -u 65532 -g nonroot nonroot
+USER 65532
+
+ENTRYPOINT ["/ko-app/pipelines-as-code-webhook"]
```

Comment:
those three Docker files are copy and paste to each others, can't be use a ARG to generate the image ?

Reply:
Not sure if `buildah` has the option ; I hope it does — if it does, we my be able to try that out (but same as commented elsewhere, we need to update our tool)

Reply:
actually we can have one dockerfile for all 3 binaries